src/fastoad/model_base/flight_point.py

```diff
@@ -175,21 +238,31 @@ def make_absolute(self, reference_point: "FlightPoint") -> "FlightPoint":
         :return: the copied flight point with no relative field.
         """
         new_point = deepcopy(self)
```

Comment:
What does absolute and relative means for fields ?

Reply:
Typically, it allows setting a target flight point in a segment that would be, e.g., 3000m above the start flight point.

Reply:
I'm still not sure I understand but we can discuss that face to face, it doesn't prevent the approval of the PR